### DISCUSSION TITLE
Make chat scroll bar scroll wheel only - not clickable...

### DISCUSSION THREAD

**Discussion by Feenoh** [runelite/runelite#15322] [Ideas]
I like being able to click thru the transparent chat box to monitor drops/PBs easily. 

**I keep clicking the damn scoll bar when I'm woox walking at vorkath and its gotten me killed a few times**

Would it be possible to turn this feature off by maybe making it so the only way to navigate chat is via the mouse scroll wheel? It's the small things bois <3

**Comment by Alexsuperfly** [comment]
you can put the scroll bar on the other side so its harder to actually click on it
